### DISCUSSION TITLE
multiple ldaps in ldap uri

### DISCUSSION THREAD

**Discussion by lassieee** [quay/quay#1031] [Q&A]
Hi,

I'm a new Quay user. I've deployed Quay on OpenShift and I'm trying to configure multiple ldap servers in the "LDAP URI" field, but this does not seem possible.
I cannot find anything about this in the docs either.

Can anyone please confirm if this is, or isn't, possible?

I'm using Quay v3.6.1 on OpenShift 4.8.10.

Thanks in advance.

Kind regards,
Casper

**Comment by lassieee** [comment]
figured it out, each ldap server needs to end with a slash '/', e.g. "ldaps://ldap01/ ldaps://ldap02/"

**Comment by dkoci** [comment]
Hi Casper. Are you using projectquay or redhat quay in openshift? I am still a bit confused about the difference. Still dont know if redhat quay operator which is available in openshift is free for use.
